packages/core/src/routes/admin-user/social.ts

```diff
@@ -154,4 +157,30 @@ export default function adminUserSocialRoutes<T extends ManagementApiRouter>(
       return next();
     }
   );
+
+  /**
+   * This route is for integration tests only.
+   * It retrieves the social identity relations of a user by their user ID.
+   */
```

Comment:
nit: a little bit strange to host an "integration test only" endpoint - maybe elaborate the reason (e.g. check data integrity for IDs)

Reply:
Add more comments to indicate the purpose of this test-only API.